src/components/TrackTypeDropdown.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+/**
+ * A searchable track type dropdown component.
```

Comment:
```suggestion
 * A track type dropdown component.
```

This isn't really meaningfully searchable.